libs/payments/events/src/lib/emitter.service.ts

```diff
@@ -43,12 +46,22 @@ export class PaymentsEmitterService {
     );
     this.emitter.on('sp3Rollout', this.handleSP3Rollout.bind(this));
     this.emitter.on('locationView', this.handleLocationView.bind(this));
+    this.emitter.on('auth', this.handleAuthEvent.bind(this));
   }
 
   getEmitter(): Emittery<PaymentsEmitterEvents> {
     return this.emitter;
   }
 
+  async handleAuthEvent(eventData: AuthEvents) {
+    const { type, errorMessage } = eventData;
+    this.statsd.increment(`auth_${type}`);
```

Comment:
Can we make the name of this static, with the type as a tag? It's hard to enumerate in promql across multiple different names, but easy to with tags.

Reply:
Oh yes for sure! Thank you, I was curious about this.